tests/services/test_rest_svc.py

```diff
@@ -65,6 +65,53 @@ def setup_rest_svc_test(loop, data_svc):
+                                            'adversary_id': 'ad-hoc', 'atomic_ordering': [], 'objective': None,
+                                            'tags': [], 'has_repeatable_abilities': False}, 'state': 'finished',
+                              'planner': {'name': 'test', 'description': None, 'module': 'test',
+                                          'stopping_conditions': [], 'params': {}, 'allow_repeatable_abilities': False,
+                                          'ignore_enforcement_modules': [], 'id': '123'}, 'jitter': '2/8',
+                              'host_group': [{'trusted': True, 'architecture': 'unknown', 'watchdog': 0,
+                                              'contact': 'unknown', 'username': 'unknown', 'links': [], 'sleep_max': 8,
+                                              'exe_name': 'unknown', 'executors': ['pwsh', 'psh'], 'ppid': 0,
+                                              'sleep_min': 2, 'server': '://None:None', 'platform': 'windows',
+                                              'host': 'unknown', 'paw': '123', 'pid': 0,
+                                              'display_name': 'unknown$unknown', 'group': 'red', 'location': 'unknown',
+                                              'privilege': 'User', 'proxy_receivers': {}, 'proxy_chain': [],
+                                              'origin_link_id': 0, 'deadman_enabled': False,
+                                              'available_contacts': ['unknown'], 'pending_contact': 'unknown'}],
+                              'visibility': 50, 'autonomous': 1, 'chain': [], 'auto_close': False,
+                              'obfuscator': 'plain-text', 'objective': {'goals': [{'value': 'complete',
+                                                                                   'operator': '==',
+                                                                                   'target': 'exhaustion',
+                                                                                   'achieved': False,
+                                                                                   'count': 1048576}],
+                                                                        'percentage': 0.0, 'description': '',
+                                                                        'id': '495a9828-cab1-44dd-a0ca-66e58177d8cc',
+                                                                        'name': 'default'}}
+        internal_rest_svc = rest_svc(loop)
+        operation = loop.run_until_complete(internal_rest_svc.create_operation(access=dict(
+            access=(internal_rest_svc.Access.RED, internal_rest_svc.Access.APP)),
+            data=dict(name='My Test Operation', planner='test', source='123', state='finished')))
+        operation_id = operation[0]["id"]
+        new_operation = loop.run_until_complete(data_svc.locate('operations', match=dict(id=operation_id)))[0].display
+        new_operation.pop('id')
```

Comment:
This looks good overall. I think my only real thought is that it might be better to add to `expected_operation` rather than take away from `new_operation` (which might be better renamed `found_operation` since it's technically the same, just pulled from memory). One way you could do this is instead of using `pop` to remove the id, add the ID from `operation_id` to `expected_operation` in it's place.